envoy-filters/mgw-source/filters/http/mgw-wasm-websocket/filter.cc

```diff
@@ -113,14 +164,13 @@ FilterHeadersStatus MgwWebSocketContext::onResponseHeaders(uint32_t, bool) {
 FilterDataStatus MgwWebSocketContext::onRequestBody(size_t body_buffer_length,
                                                bool /* end_of_stream */) {
   auto body = getBufferBytes(WasmBufferType::HttpRequestBody, 0, body_buffer_length);
-  LOG_TRACE(std::string("onRequestBody called mgw_WASM_websocket") + std::string(body->view()));
+  LOG_TRACE(std::string("onRequestBody called mgw_WASM_websocket ") + std::string(body->view()) + std::string(" : ") + this->x_request_id_ );
```

Comment:
shall we not include the body->view all together? I don't think we need to print the payload.